packages/inventory-compliance/src/SystemRulesTable.js

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 import propTypes from 'prop-types';
 import { Pagination, PaginationVariant, ToolbarItem } from '@patternfly/react-core';
-import { CheckCircleIcon, ExclamationCircleIcon } from '@patternfly/react-icons';
-import { Table, TableHeader, TableBody } from '@patternfly/react-table';
+import { CheckCircleIcon, ExclamationCircleIcon, AnsibeTowerIcon as Ansible } from '@patternfly/react-icons';
```

Comment:
What's the rationale on renaming the icon name?

Reply:
... evening musings. :D I removed the alias.